CODE_OF_CONDUCT.md

```diff
@@ -1,21 +1,16 @@
-# Code of Conduct
+# The Creator's Code
 
-We are sentient.  
-To be sentient is to be limited.  
-In our limitation, we make choices that are unwise or are flawed.
+Because we are limited beings,
+we make choices that are unwise or flawed.
 
-If we make unwise choices because of our limitation,  
-we cannot judge others for the same reason.
+While others may meet us with hostility,
+we choose to acknowledge difference:
 
-So, we cannot judge,  
-thus we forgive. 
+Thus we forgive. 
```

Comment:
This is so important and makes a great conclusion to the antistrophe. It has impact, it's simply yet powerfully put and it captures the essence of this text (for me).

That's why I've put it on its own line in its own section (although it's still very much part of the antistrophe). :-)